src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -45,7 +45,30 @@
     "7229D034-7DBE-4FD7-B0E4-38D617571F93", // UITests
     "E2821155-50B7-468D-A15B-E0EAF55DAC2B", // WINDOWS - Package.appxmanifest Identity.Name
   ],
-  "symbols": { 
+  "symbols": {
+    "appId": {
+      "type": "parameter",
+      "datatype": "string",
+      "replaces": "com.companyname.MyExtensionsApp",
```

Comment:
keep this all lowercase (and make sure it's all lowercase in the manifest/plist files)

Reply:
that's what was in the manifest files... I updated the info.plist/android.manifest but didn't get that one... good catch